main branch builds broken when sentry feature is enabled after 789 PR merge

Oops, merging the https://github.com/imageworks/spk/pull/789 PR messed up the builds. We're seeing these errors when building the `main` branch with the `sentry` feature now:
```
error[E0425]: cannot find value `render_summary` in this scope
   --> crates/spfs-cli/cmd-enter/src/cmd_enter.rs:261:16
    |
261 |             if render_summary.is_zero() {
    |                ^^^^^^^^^^^^^^ help: a local variable with a similar name exists: `_render_summary`
```
There are a bunch of these errors because of the parameter rename in the enclosing function. I guess it was missed because the `main` branch doesn't get built or tested with that feature enabled?

--- a/crates/spfs-cli/cmd-enter/src/cmd_enter.rs
+++ b/crates/spfs-cli/cmd-enter/src/cmd_enter.rs
@@ -245,7 +245,7 @@ impl CmdEnter {
     }
 
     #[cfg_attr(not(feature = "sentry"), allow(unused))]
-    fn report_render_summary(&self, _render_summary: RenderSummary, render_time: f64) {
+    fn report_render_summary(&self, render_summary: RenderSummary, render_time: f64) {
         if self.enter.metrics_in_env {
             // The render time is put into environment variables for
             // other, non-spfs, programs to access. If this command

